packages/dev/core/src/Cameras/Inputs/arcRotateCameraGamepadInput.ts

```diff
@@ -97,25 +97,25 @@ export class ArcRotateCameraGamepadInput implements ICameraInput<ArcRotateCamera
 
             if (rsValues) {
                 if (rsValues.x != 0) {
-                    const normalizedRX = rsValues.x / this.gamepadRotationSensibility;
-                    if (normalizedRX != 0 && Math.abs(normalizedRX) > 0.005) {
-                        camera.inertialAlphaOffset += normalizedRX;
+                    const normalizedRx = rsValues.x / this.gamepadRotationSensibility;

```

Comment:
RX is not an initialism or anything really. They are unrelated. I think lowercasing the X and Y makes the code confusing. Maybe spell out the Right and Left?

Reply:
Of course, that can be changed